README.md

```diff
@@ -2,33 +2,69 @@
 
 The idea behind this was to make a simple and functional rewrite of plex debrid that seemed to get a bit clustered.
 
-Rewrite of plex_debrid project, limited functionality:
-- Services include: plex, mdblist, torrentio and realdebrid
+Rewrite of [plex_debrid](https://github.com/itsToggle/plex_debrid) project.
+
+Currently:
+- Services include: Plex, Mdblist, Torrentio and Real Debrid
 
 TODO:
+- Implement uncached download in real-rebrid, dont know if we need this, movies seem to work ok...
+- Implement updating quality of fetched items if below something
+- Add frontend, ongoing... (adding api endpoints as we go along)
+
+Check out out [Project Board](https://github.com/users/dreulavelle/projects/2) to stay informed!
+
+COMPLETED:
+- ~~Update plex libraries for changes, ongoing... ~~; (functional but we need to be more specific when to update)
 - ~~Real-debrid should download only one file per stream, lets avoid collections~~
-- ~~Modify scraping logic to try scaping once a day if not found?~~
 - ~~Add overseerr support, mostly done~~; still need to mark items as available?
+- ~~Add support for shows, ongoing...~~ (Functionalish, needs work...)
+- ~~Modify scraping logic to try scaping once a day if not found?~~
 - ~~Store data with pickle~~
 - ~~Improve logging...~~
-- ~~Update plex libraries for changes, ongoing... ~~; (functional but we need to be more specific when to update)
-- Add frontend, ongoing... (adding api endpoints as we go along)
-- ~~Add support for shows, ongoing...~~ (Functionalish, needs work...)
-- Implement uncached download in real-rebrid, dont know if we need this, movies seem to work ok...
-- Implement updating quality of fetched items if below something
+- And more..
+
+Please add features and mention issues over on our [Issue Tracker](https://github.com/dreulavelle/iceberg/issues)!
 
+We are constantly adding features and improvements as we go along and squashing bugs as they arise.
 
-## Running the project
+Enjoy!
 
+## Docker Compose
+
+```yml
+version: '3.8'
+
+services:
+  iceberg:
+    image: iceberg:latest
+    container_name: Iceberg
+    ports:
+      - "4173:4173"
+    volumes:
+      - ./logs:/iceberg/logs
+      - ./settings.json:/iceberg/settings.json
 ```
+
+> [!WARNING]  
```

Comment:
Yes, but you can't bind mount a file that isn't there. And if you bind mount to an empty file, it creates issues.

Reply:
The best way arround this is to bind to a directory.. so if we tosses settings.json into a `settings/settings.json` and we binded to `settings` dir. Then it would work fine.